src/swoole/tests/Unit/RuntimeTest.php

```diff
@@ -15,7 +15,7 @@ class RuntimeTest extends TestCase
 {
     public function testGetRunnerCreatesARunnerForCallbacks(): void
     {
-        $options = [];
+        $options = ['error_handler' => false];
```

Comment:
This change was needed due to `GenericRuntime` introducing an error handler replacement by default.

Luckily, PHPUnit catches this sort of mistake, so it was quite trivial to find the root cause.